edx_proctoring/backends/software_secure.py

```diff
@@ -114,11 +112,25 @@ def pad(text):
         encrypted_text = cipher.encrypt(pad(pwd))
         return base64.b64encode(encrypted_text)
 
-    def _get_payload(self, exam, time_limit_mins, attempt_code,
-                     is_sample_attempt, callback_url):
+    def _get_payload(self, exam, context):
         """
         Constructs the data payload that Software Secure expects
         """
+
+        attempt_code = context['attempt_code']
+        time_limit_mins = context['time_limit_mins']
+        is_sample_attempt = context['is_sample_attempt']
+        callback_url = context['callback_url']
+        full_name = context['full_name']
+        first_name = ''
+        last_name = ''
+
+        if full_name:
+            name_elements = full_name.split(' ')
+            first_name = name_elements[0]
+            if len(name_elements) > 1:
+                last_name = ' '.join(name_elements[1:])
```

Comment:
Yea, Software Secure's API only supports a first_name, last_name field. It's used by reviewers to compare the name that will be printed on the edX Certificate of completion to the photo ID that the user presented as part of their proctoring startup.